component/file_cache/file_cache.go

```diff
@@ -651,6 +651,46 @@ func (fc *FileCache) IsDirEmpty(options internal.IsDirEmptyOptions) bool {
 	return fc.NextComponent().IsDirEmpty(options)
 }
 
+// DeleteEmptyDirs: delete empty directories in local cache, return error if directory is not empty
+func (fc *FileCache) DeleteEmptyDirs(options internal.DeleteDirOptions) (bool, error) {
```

Comment:
Why do we have 2 return parameters here if they both signify the same thing!

Reply:
You can refer the previous comment. It's related to why I updated this method to return two values.